### PR TITLE
Fix memo serialization.

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
### ❓ Context

- **Impacted projects**:
Helium ledger integration - `libs/ledgerjs/packages/hw-app-helium`

Fixes incorrect serialization of memo field which was causing transaction verification to fail.

### ✅ Checklist

- [x] **Test coverage**: _Did you write any tests to cover the changes introduced by this pull request?_
- [x] **Atomic delivery**: _Is this pull request standalone? In order words, does it depend on nothing else?_
- [x] **No breaking changes**: _Does this pull request contain breaking changes of any kind? If so, please explain why._

### CODE REVIEW

File: libs/ledgerjs/packages/hw-app-helium/package.json [hunk 1]
```diff
@@ -29,16 +29,17 @@
...
     "@helium/proto": "^1.5.0",
-    "@helium/transactions": "^3.62.0",
+    "@helium/transactions": "^4.1.0",
     "@ledgerhq/errors": "workspace:^",
     "@ledgerhq/hw-transport": "workspace:^",
     "@ledgerhq/logs": "workspace:^",
     "bignumber.js": "^9.0.1",
-    "bip32-path": "^0.4.2"
+    "bip32-path": "^0.4.2",
+    "long": "^5.2.0"
```
Comment (haammar-ledger) [inline libs/ledgerjs/packages/hw-app-helium/package.json:38]: Hi @matthewcarlreetz is this library safe in React Native environment?

Comment (matthewcarlreetz) [reply libs/ledgerjs/packages/hw-app-helium/package.json:38]: Yes, I have tested it using React Native

**Review by haammar-ledger — APPROVED** [review]

**Comment by haammar-ledger** [comment]
@matthewcarlreetz looks good to me, can you please just solve the conflict.

**Comment by matthewcarlreetz** [comment]
@haammar-ledger conflicts resolved ✔️

**Comment by haammar-ledger** [comment]
Hi @matthewcarlreetz sorry about the delay I was off last week. :slightly_smiling_face:

**Pull request merged**